### DISCUSSION TITLE
麻烦问一下为什么不能选择到gpt-40，后台模型我是开放的

### DISCUSSION THREAD

**Discussion by zhangleijun111** [lobehub/lobehub#2810] [General Question] [answered]
### 📦 Environment

- [ ] Official
- [ ] Official Preview
- [ ] Vercel / Zeabur / Sealos
- [X] Docker
- [ ] Other

### 📌 Version

v0.162.11

### 💻 Operating System

- [X] Windows
- [ ] macOS
- [ ] Ubuntu
- [ ] Other Linux
- [ ] iOS
- [ ] Android
- [ ] Other

### 🌐 Browser

- [X] Chrome
- [ ] Edge
- [ ] Safari
- [ ] Firefox
- [ ] Other

### 🐛 Bug Description

麻烦问一下为什么不能选择到gpt-40，后台模型我是开放的

![1717133555118](https://github.com/lobehub/lobe-chat/assets/169867372/9aa8cd4e-28ce-4240-829f-a3bd688e465c)

### 📷 Recurrence Steps

麻烦问一下为什么不能选择到gpt-40，后台模型我是开放的

### 🚦 Expected Behavior

麻烦问一下为什么不能选择到gpt-40，后台模型我是开放的

[a comment by an automated account is not shown]

**Comment by Zach677** [comment]
更新一下模型列表试试？

[a comment by an automated account is not shown]

**Comment by zhangleijun111** [comment]
> 更新一下模型列表试试？
更新过了只能获取14个模型

[a comment by an automated account is not shown]

**Comment by Zach677** [comment]
emmm
![CleanShot 2024-05-31 at 15 02 17](https://github.com/lobehub/lobe-chat/assets/104423295/7971bfb5-2c10-4d25-9523-aa24b9f9ea28)

**Comment by zhangleijun111** [comment]
你的接口地址是什么为什么我只有14个
![Uploading 1717139185193.png…]()

[a comment by an automated account is not shown]

**Comment by Zach677** [comment]
中转的

[a comment by an automated account is not shown]

**Comment by zhangleijun111** [comment]
> 中转的

所以第api地址的问题是嘛，我用的这个https://api.openai-hk.com/v1

**Comment by sxjeru** [comment, marked as answer]
可以先问问接口服务商客服，是否有提供 gpt-4o。
也可以自定义一个模型试试能不能用。

[a comment by an automated account is not shown]
